### PR TITLE
Sample SettingsPage.OnNavigatedTo() defends against calling ToString …

### PR DESCRIPTION
**Pull request by angrah**
…on a null e.Parameters, but then calls ToString() on the result of Deserialize(null) which is also null.

A recent change removed the parameter from the navigate in Shell.xaml

### CODE REVIEW

File: Templates (Project)/Hamburger/Views/SettingsPage.xaml.cs [hunk 1]
```diff
@@ -17,8 +17,11 @@ public SettingsPage()
 
         protected override void OnNavigatedTo(NavigationEventArgs e)
         {
-            var index = int.Parse(_SerializationService.Deserialize(e.Parameter?.ToString()).ToString());
-            MyPivot.SelectedIndex = index;
+            if (e.Parameter != null)
+            {
+                var index = int.Parse(_SerializationService.Deserialize(e.Parameter.ToString()).ToString());
```
Comment (JerryNixon) [inline Templates (Project)/Hamburger/Views/SettingsPage.xaml.cs]: This does not, however, defend against Deserialize returning null.

Comment (angrah) [reply Templates (Project)/Hamburger/Views/SettingsPage.xaml.cs]: That's true, but it didn't before this change either.

Comment (angrah) [reply Templates (Project)/Hamburger/Views/SettingsPage.xaml.cs]: The JsonSerializationService doesn't return null unless you feed it null or it got null from Json.DeserializeObject(). Better to be safe I guess you're right. I will add the extra elvis and update this pull request.

**Comment by angrah** [comment]
Some of these changes already snuck in with my other pull request. Still getting the hang of this git thing.

**Pull request merged**